src/praisonai-agents/praisonaiagents/memory/adapters/factories.py

```
@@ -304,6 +304,16 @@ def _sanitize_metadata(self, metadata: Dict) -> Dict:
                 sanitized[k] = str(v)
         return sanitized
+    
+    def close(self):
+        """Clean up ChromaDB resources."""
+        try:
+            # ChromaDB PersistentClient doesn't have explicit close,
+            # but we should release references
+            self.client = None
+            self.collection = None
+        except Exception:
+            pass  # Ignore cleanup errors
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)